packages/office-ui-fabric-react/src/components/TeachingBubble/docs/new/TeachingBubbleBestPractices.md

```diff
@@ -0,0 +1,15 @@
+### Layout
+
+- TeachingBubbles can be used in sequence to walk people through complex, multistep interactions only. And only show one TeachingBubble at a time.
+  - A maximum of no more than 3 sequenced TeachingBubbles should be used in a single experience.
+  - Sequenced TeachingBubbles should have “x of y” text to indicate progress through the sequence. For example, the first TeachingBubble in a sequence might be “1 of 3”.)
+  - The action button on a sequenced TeachingBubble should read “Next” and the final sequenced TeachingBubble should have an action button with text “Got it”, that closes the experience.
+- Always place the primary button on the left, the secondary button just to the right of it.
+- Show only one primary button that inherits theme color at rest state. In the event there are more than two buttons with equal priority, all buttons should have neutral backgrounds.
```

Comment:
Aren't the colors typically inverted within a teaching bubble? (for example blue background, white text, white primary button with blue text)